mermaid_hook.js

```diff
@@ -17,7 +17,10 @@ const patch_fs = (fs, volume, methods = []) => {
   };
 };
 
-process.on("message", (volumeJSON) => {
+process.on("message", function listener(volumeJSON) {
+  // remove the current listener so process will exit when mermaid is done
+  process.removeListener("message", listener);
```

Comment:
Good point!
Could also have used `process.once(...)` instead. But I did not think about this.
👍

Reply:
Ahhh, I didn't even know `process.once()` existed, I'm too used to using Promises :)

I'll rebase my commits to use `process.once()`.